### ISSUE TITLE
Group Capstone Project

### ISSUE THREAD

**Issue by chepkok3** [chepkok3/Module-2-Capstone-Project#21]
### Capstone Project ### 

- [x] https://github.com/chepkok3/Module-2-Capstone-Project/projects/2
- [x] We are two in the group working on the capstone project.

**Comment by pinkmoon25** [comment]
Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media0.giphy.com/media/qLWdMYX1NYF2g/giphy-downsized-medium.gif"/>

## Highlights
Template board used :heavy_check_mark: 

## Required Changes ♻️
**Good work using the correct template :clap: , However, please notice all the tasks in your kanban board are unassigned! :point_down:** 

![image](https://user-images.githubusercontent.com/97521557/218518010-70eeaec4-b8f3-430b-bd35-a146ed1eb04f.png)

**I strongly recommend you to assign each tasks to each individual who is working on it as it is a core requirement that all cards are assigned to team members. You should see a circle with the user's avatar on each card. See the example below :point_down:** 

![image](https://user-images.githubusercontent.com/97521557/218496589-f1f2c0a6-d6c8-4d97-b196-26f3b4f665a1.png)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chepkok3** [comment]
@PinkMoon25 Can you please guide us on how to do that? We tried but couldn't get it right.

**Comment by wollfishTushar** [comment]
Hi @chepkok3 , I think you have already done it, you just needed to assign those tasks to each other

**Comment by pinkmoon25** [comment]
Hi @chepkok3 , I think you have already done it, you just needed to assign those tasks to each other

**Comment by chepkok3** [comment]
@PinkMoon25 Yes

**Comment by pinkmoon25** [comment]
Hi @chepkok3 please submit again for a re review and once it is approved by the next reviewer you can close the issue and move forward with your projects 👍

**Comment by chepkok3** [comment]
@PinkMoo It was already approved because the dashboard showed that it is complete.
